### PR TITLE
[shuffle] shuffle transactions fixed bug

### PR DESCRIPTION
**Pull request by avin1100**
## Motivation

After I rebased, transactions was failing due to this PR: #9571. I was using a key that didn't exist in the updated json. This PR fixes transactions 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/diem/diem/tree/main/developers.diem.com, and link to your PR here.)

## If targeting a release branch, please fill the below out as well

 * Justification and breaking nature (who does it affect? validators, full nodes, tooling, operators, AOS, etc.)
 * Comprehensive test results that demonstrate the fix working and not breaking existing workflows.
 * Why we must have it for V1 launch.
 * What workarounds and alternative we have if we do not push the PR.

### CODE REVIEW

File: shuffle/cli/src/transactions.rs [hunk 1]
```diff
@@ -123,7 +122,7 @@ fn parse_txn_for_seq_num(last_txn: &Value) -> Result<i64> {
 fn parse_json_for_account_seq_num(json_objects: Vec<Value>) -> Result<u64> {
     let mut seq_number_string = "";
     for object in &json_objects {
-        if object["type"]["name"] == "DiemAccount" {
+        if object["type"] == "0x1::DiemAccount::DiemAccount" {
```
[the remark this answers is not in the document]
Comment (avin1100) [reply shuffle/cli/src/transactions.rs:125]: Yup will do.

**Review by xli — APPROVED** [review]

**Review by dimroc — CHANGES_REQUESTED** [review]
Would like to investigate consts on the Dev API side of things so we're not siloed from changes in that crate.

**Review by dimroc — DISMISSED** [review]
No existing use of `0x1::DiemAccount::DiemAccount` in the api/src side of things. Approving.

**Pull request merged**